cmd/fluxd/main.go

```diff
@@ -145,6 +148,12 @@ func main() {
 		}
 	}
 
+	switch {
```

Comment:
Took a (brief) moment to grok this with context, so for contextless future sam I think the intent would be more clearly expressed here with a plain if statement.

Reply:
Oh no github, we couldn't possibly want to see the lines _after_ this one, don't be silly...

Reply:
I don't know it seemed simpler to write it as a `switch`.